share/completions/pabcnet_clear.fish

```diff
@@ -1 +0,0 @@
-complete -f -c pabcnet_clear -a '/Help\t"Display help" /H\t"Display help" /\?\t"Display help" /Debug\t"Generate code with optimizations" /output\t"Executable name" /SearchDir\t"Add path to unit search directories"'
```

Comment:
Okay, I'm confused. Is this tool called "pabcnet_clear" or "pbcnetcclear"?

Reply:
The second option is right.